From 257d1f042c2f1b17e36b8a20bc8fd34b2552abcc Mon Sep 17 00:00:00 2001
From: Jan Kiszka <jan.kiszka@siemens.com>
Date: Sun, 4 Jun 2023 18:21:04 +0200
Subject: [PATCH 1/5] meta-isar: u-boot-starfive-visionfive2: Fix fetching,
 update to BSP version VF2_v3.0.4

JH7110_VisionFive2_devel is apparently rebasing, and now it left
our previously chosen commit orphaned behind. Update to latest BSP
U-Boot release and avoid that we may be surprised again.

Signed-off-by: Jan Kiszka <jan.kiszka@siemens.com>
---
 ....5.bb => u-boot-starfive-visionfive2_2021.10+VF2-3.0.4.bb} | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)
 rename meta-isar/recipes-bsp/u-boot/{u-boot-starfive-visionfive2_2022.01-rc4+VF2-2.11.5.bb => u-boot-starfive-visionfive2_2021.10+VF2-3.0.4.bb} (85%)

diff --git a/meta-isar/recipes-bsp/u-boot/u-boot-starfive-visionfive2_2022.01-rc4+VF2-2.11.5.bb b/meta-isar/recipes-bsp/u-boot/u-boot-starfive-visionfive2_2021.10+VF2-3.0.4.bb
similarity index 85%
rename from meta-isar/recipes-bsp/u-boot/u-boot-starfive-visionfive2_2022.01-rc4+VF2-2.11.5.bb
rename to meta-isar/recipes-bsp/u-boot/u-boot-starfive-visionfive2_2021.10+VF2-3.0.4.bb
index 0eeedc1a..1be1080e 100644
--- a/meta-isar/recipes-bsp/u-boot/u-boot-starfive-visionfive2_2022.01-rc4+VF2-2.11.5.bb
+++ b/meta-isar/recipes-bsp/u-boot/u-boot-starfive-visionfive2_2021.10+VF2-3.0.4.bb
@@ -5,8 +5,8 @@
 
 require recipes-bsp/u-boot/u-boot-custom.inc
 
-SRC_URI += "git://github.com/starfive-tech/u-boot.git;branch=JH7110_VisionFive2_devel;protocol=https;destsuffix=u-boot-${PV}"
-SRCREV = "ac0ac696256abf412826d74ee918dd417e207d7b"
+SRC_URI += "git://github.com/starfive-tech/u-boot.git;nobranch=1;protocol=https;destsuffix=u-boot-${PV}"
+SRCREV = "b6e2b0e85c774a18ae668223a6e5f7d335895243"
 
 DEBIAN_BUILD_DEPENDS .= ", libssl-dev:${DISTRO_ARCH}"
 # when cross compiling, we need the library on the host as well, as the signature computation is done locally
-- 
2.34.1

